adapters/repos/db/repo.go

```diff
@@ -248,10 +251,21 @@ func (db *DB) worker(first bool) {
 	}
 }
 
-type job struct {
+type batchJob struct {
 	object  *storobj.Object
 	status  objectInsertStatus
 	index   int
 	ctx     context.Context
 	batcher *objectsBatcher
 }
+
+type bM25fJob struct {
```

Comment:
This would also include hybrid, right? Would it make sense to have the queue "lower" in the stack so only the bm25-part of hybrid search would go in here?